enigma-principal/enclave/src/epoch_keeper_t/nested_encoding.rs

```diff
@@ -15,25 +15,25 @@ impl NestedSerialization for U256 {
         let mut msg = [0u8;32];
 
         self.to_big_endian(&mut msg);
-        let len = msg.len().to_be_bytes();
+        let len = (msg.len() as u64).to_be_bytes();
 
         res.push(ZERO);
         res.extend_from_slice(&len);
         res.extend_from_slice(&msg);
-        (msg.len(), res)
+        (res.len(), res)
```

Comment:
I think that now it is redundant to return the length of `res`, since `res.len()` always may be done.